docs/src/main/paradox/starvation-detector.md

```diff
@@ -49,6 +49,12 @@ You can customize settings of the starvation detector to prevent spurious loggin
 By default, the starvation detector runs seldom enough not to cause any performance hit itself. Thread starvation issues usually affect
 systems for longer time spans, so the starvation detector is still likely to experience and warn even when it runs only infrequently.
 
+@@@ note { title="Java 17" }
+
+When using the Starvation Detector with Java 17 you have to add JVM flag `--add-opens=java.base/java.util.concurrent=ALL-UNNAMED`.
```

Comment:
what happens if this is not done? Would be good if we can catch the error, log a warning and then continue without running Starvation Detector

Reply:
This is a really good question. In trying to answer it I just found that the Starvation Detector doesn't detect a simple `Thread.sleep` in a hello world sample. I'll keep debugging to see what's going wrong.

Reply:
never mind

Reply:
Users now will find the following when not setting `--add-opens ...`: 
```
2023-01-11 18:30:02,958 WARN  [StarvationDetector$StarvationDetectorThread] - Stopping Starvation detector. Reason: Unable to make field final java.util.concurrent.ForkJoinPool$ForkJoinWorkerThreadFactory java.util.concurrent.ForkJoinPool.factory accessible: module java.base does not "opens java.util.concurrent" to unnamed module @51376efb.
Probably you are missing some JVM parameters. See 'note' in https://doc.akka.io/docs/akka-diagnostics/current/starvation-detector.html#configuration
```